pkgs/by-name/un/unbound/package.nix

```diff
@@ -140,7 +140,7 @@ stdenv.mkDerivation (finalAttrs: {
     # Build libunbound again, but only against nettle instead of openssl.
     # This avoids gnutls.out -> unbound.lib -> lib.getLib openssl.
     ''
-      configureFlags="$configureFlags --with-nettle=${nettle.dev} --with-libunbound-only"
+      appendToVar configureFlags "--with-nettle=${nettle.dev} --with-libunbound-only"
```

Comment:
Couldn’t this be set outside of Bash?

Reply:
Uh, weird, okay! (Do we want to rely on the space‐splitting behaviour here, though? It doesn’t feel very structured. Maybe one day we’ll have store paths with spaces in them…)